index.html

```diff
@@ -20,6 +20,6 @@
     <script src="scripts/app.js"></script>
     <script src="scripts/controllers/main.js"></script>
     <script src="scripts/controllers/message.js"></script>
-    <script src="scripts/services/MessageService.js"></script>
+    <script src="scripts/services/messageService.js"></script>
```

Comment:
For some reason, it wouldn't let me add the renamed file to the commit. the service file needs to match this capitalization.